src/CarbonAware.Aggregators/test/ForecastAggregatorTests.cs

```diff
@@ -0,0 +1,277 @@
+﻿using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.Aggregators.Forecast;
+using CarbonAware.Aggregators.Tests;
+using CarbonAware.Interfaces;
+using CarbonAware.Model;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CarbonAware.Aggregator.Tests;
```

Comment:
Just confirming, none of these tests are functionally new right? Just moved to live within their own file?

Reply:
correct!